python/UniverseWrapper.cpp

```diff
@@ -444,7 +444,7 @@ namespace FreeOrionPython {
                                                     return_value_policy<return_by_value>()
                                                 ))
 
-            .def("dump",                        &DumpObjects)
+            .def("dump",                        &DumpObjects, return_value_policy<return_by_value>(), "Returns string with debug information, use '0' as argument.")
```

Comment:
https://github.com/freeorion/freeorion/blob/master/python/UniverseWrapper.cpp

`DumpObjects` is different from standard content and object `Dump` functions.

Reply:
> DumpObjects is different from standard content and object Dump functions.

Can you provide more clear instructions?

Reply:
Look at the signature of the functions, don't assume they're all the same because they have Dump in the name....

Reply:
I will just rollback this.